axelor-base/src/main/java/com/axelor/apps/base/service/print/PrintTemplateServiceImpl.java

```diff
@@ -104,8 +103,7 @@ public Print generatePrint(Long objectId, PrintTemplate printTemplate)
     LOG.debug("");
 
     Locale locale =
-        Optional.ofNullable(printTemplate.getLanguage())
-            .map(Language::getCode)
+        Optional.ofNullable(printTemplate.getLocalization().getCode())
```

Comment:
Keep the .map(Localization::getCode), otherwise there is no point to use the Optional.ofNullable, you'll get an NPE if localization is null